extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueBridgeHandler.java

```diff
@@ -519,4 +519,13 @@ private boolean isEqual(State state1, State state2) {
 
         return configStatusMessages;
     }
+
+    @Override
+    public String getDeviceId() {
+        try {
+            return bridge.getFullConfig().getConfig().getMACAddress().replaceAll(":", "").toLowerCase();
+        } catch (Exception e) {
```

Comment:
see above, I cannot use the value from the handler config. I agree, that it is not the best idea to catch a possible NPW, but in this particular case I would otherwise need to guard against any possible NPE and doing the same like in the exception, so why bother? Just catch die NPE looks cleaner to me.